AsyncSwift/Views/EventDetailView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  EventDetailView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct EventDetailView: View {
+
+    let data = EventModel.mockData
+
+    var body: some View {
+
+        ScrollView {
+            VStack(alignment: .leading) {
+                Divider()
+                    .padding(.top, 10)
+                VStack(alignment: .leading, spacing: 8) {
+                    Text(data.event.subject)
+                        .fontWeight(.bold)
+                        .font(.system(size: 24))
+                    ForEach(data.event.description, id:\.self) { paragraph in
+                        Text(paragraph.content)
+                            .font(.system(size: 17))
```

Comment:
font size를 시스템 폰트크기설정한 유저를 고려하여 폰트 이름을 직접 사용해주세요!
```suggestion
                            .font(.body))
```